code/datums/extensions/cell/cell.dm

```diff
@@ -0,0 +1,158 @@
+/datum/extension/loaded_cell
+	expected_type = /obj/item
```

Comment:
Is the /item restriction required?

Reply:
Load and unload are on the extension though, so someone could use it with a structure and just add the interaction through something like attackby. Either way works though.